alphastats/gui/pages/04_Analysis.py

```diff
@@ -44,18 +48,27 @@
 
 c1, c2 = st.columns([0.33, 0.67])
 with c1:
+    plotting_options = PlottingOptions.get_values()
     method = st.selectbox(
         "Analysis",
         options=["<select>"]
         + ["------- plots -------"]
-        + list(get_plotting_options(st.session_state).keys())
+        + plotting_options
+        + [
+            key
+            for key in list(get_plotting_options(st.session_state).keys())
```

Comment:
dumb question: why do we have a separate `plotting_options` thing again? why not just have the output of `get_plotting_options`? because we anyways have both here?

Reply:
UPD: I see it's gone in the next one